go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
-	github.com/cloudflare/cloudflare-go v0.60.0
```

Comment:
you can remove this version bump. we have automation that does this and once the new version lands, this PR can just be synced.